Library/Formula/duck.rb

```diff
@@ -0,0 +1,29 @@
+class Duck < Formula
+  homepage "https://duck.sh/"
+  url "https://dist.duck.sh/duck-src-16336.tar.gz"
+  version "4.6.2.16336"
```

Comment:
Sorry, I mean if someone other than you is updating this package how do they know what to set as the new `version`?